lints.rs

```diff
@@ -13,36 +13,45 @@
     clippy::exit,
     clippy::explicit_into_iter_loop,
     clippy::filter_map_next,
+    clippy::float_cmp_const,
     clippy::fn_params_excessive_bools,
+    clippy::get_unwrap,
     clippy::if_let_mutex,
     clippy::imprecise_flops,
     clippy::inefficient_to_string,
     clippy::let_unit_value,
     clippy::linkedlist,
     clippy::lossy_float_literal,
     clippy::macro_use_imports,
+    clippy::map_err_ignore,
```

Comment:
`map_err_ignore` we've done extensive testing of and as of today switched over all of our code to use, for the same reason you mentioned also of enforcing not loosing error context, or having to drop error context explicitly with a named unused variable (`|_err|` instead of `|_|`). so this plan to add to our set!